packages/frontend/src/features/sqlRunner/components/visualizations/Table.tsx

```diff
@@ -25,7 +25,7 @@ export const Table: FC<Props> = ({ data }) => {
         (state) => state.sqlRunner.resultsTableConfig,
     );
     const {
-        tableContainerRef,
+        tableWrapperRef,
```

Comment:
renamed to avoid confusion. 
The `TableContainer` will group everything and provide styling
The `Flex`/wrapper will receive the table stuff